ios/FluentUI/Avatar/AvatarTokens.swift

```diff
@@ -6,179 +6,274 @@
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark),
+                .init(light: globalTokens.brandColors[.shade30].light, dark: globalTokens.brandColors[.tint40].dark)]
+    }
 
+    /// The default color of the foreground of the `Avatar`
+    open var foregroundDefaultColor: DynamicColor {
         switch style {
-        case .default:
-            appearanceProxy = currentTheme.MSFAvatarTokens
+        case .default, .group:
+            return .init(light: globalTokens.brandColors[.primary].light, dark: globalTokens.neutralColors[.black])
         case .accent:
-            appearanceProxy = currentTheme.MSFAccentAvatarTokens
+            // this was $Icon.accentColor, but we don't seem to have that. icon.accent color was just black and white, though
+            return .init(light: globalTokens.neutralColors[.white], dark: globalTokens.neutralColors[.black])
```

Comment:
Was there a better choice for the color to return, rather than just remake Icon.accentColor?